src/java.base/windows/native/libjava/java_props_md.c

```diff
@@ -321,6 +321,26 @@ SetupI18nProps(LCID lcid, char** language, char** script, char** country,
     return TRUE;
 }
 
+// For dynamic lookup of GetTempPath2 API
+typedef DWORD (WINAPI *GetTempPath2WFnPtr)(DWORD, LPWSTR);
+static GetTempPath2WFnPtr _GetTempPath2W = NULL;
```

Comment:
I think the code should use "nullptr" instead of NULL? I'm OK with it being "NULL" if other places in the file also uses NULL.

Reply:
Why all these leading "_" ? Is that a coding guideline for that?

Reply:
Using NULL as it is C code and others are C++.